### DISCUSSION TITLE
카카오 질문드립니다.

### DISCUSSION THREAD

**Discussion by THePhanT00M** [kimytsc/covid-rest-vaccine-macro#73] [Q&A]
나와의 채팅을 열고, https://accounts.kakao.com/login?continue=https%3A%2F%2Fvaccine-map.kakao.com%2Fapi%2Fv1%2Fuser 메세지를 먼저 남겨둡니다. (여기까지 문제 없음)
아래의 소스를 복사하고 카카오 브라우저 주소창에 코드를 붙혀넣기 했는데 네이버 잔여백신 지도로 넘어가는데 무슨 문제일까요?

**Comment by kimytsc** [comment]
카카오톡 9.4.6 버전(최신)에서 코드를 그대로 복사해서 테스트해보니 정상 동작 확인했습니다.

혹시 복사해서 붙여넣은 텍스트가 지도URL만 노출었는지 확인 가능할까요?

**Reply by THePhanT00M** [reply]
주소창에
javascript:((my={
    map: "https://m.place.naver.com/rest/vaccine?vaccineFilter=used&x=127.0572402&y=37.5143501&bounds=126.9748428%3B37.4847631%3B127.1396377%3B37.5439254",
    delay: 1000,
    timeout: 3000,
    choice: [
        "VEN00013",
        "VEN00014"
    ]
}) => {
fetch('https://raw.githubusercontent.com/kimytsc/covid-rest-vaccine-macro/main/kakao/macro.js')
.then(res => res.text())
.then(res => {
    var d=document
    , s=d.createElement('script');
    Object.keys(my).forEach(k=>{s.setAttribute(k, my[k])});
    s.innerHTML=res;
    d.getElementsByTagName('head')[0].appendChild(s);
});
})()
해당코드를 넣어더니 네이버 백신 지도 페이지로 넘어갔습니다,.

**Comment by THePhanT00M** [comment]
아이폰 카카오톡 브라우저에서는 javascript:alert(1); 해도 검색어로 인식을 하는거 같습니다
안드로이드 카카오톡 브라우저에서는 javascript:alert(1); 하면 스크립트가 작동을 하는거 같습니다,.

**Reply by kimytsc** [reply]
아.. 아이폰ㅠㅠ 해당 현상 확인해보겠습니다. 감사합니다!

**Comment by EddieK214** [comment]
개발자님 우선 이렇게 도움주셔서 감사합니다 ㅎㅎ 저도 위와같은 증상이 있네용,, 아이폰으로 진행중인데 아직 방법이 없는걸까요,,? ㅎㅎ 감사합니당

**Reply by kimytsc** [reply]
아직 아이폰으로는 해결 방법을 못찾은 상태입니다... 네이버를 이용해서 진행해주시면 될 것 같습니다.

**Reply by EddieK214** [reply]
아아 넵 알겠습니다 ㅎㅎ 감사합니다!!!!!!

**Reply by SJang1** [reply]
아이폰.. 애플이 사파리에서 자바스크립트 실행을 막아둔 걸로 보여집니다.
![IMG_5352](https://user-images.githubusercontent.com/29011440/130163135-f4d925fc-5a9d-426b-9a39-abeeb35c128d.jpeg)
Shortcuts를 이용하는 방법이 하나 있을걸로 보여지는데 이상하게 작동이 안되어서 머리 싸메고 있네요.
그 외에도 북마크를 사용하는 방법이 하나 있는데, 그거는 카카오톡 인앱브라우저가 북마크를 지원하지 않아서 안되고요..
